core/src/main/java/org/elasticsearch/index/query/MoreLikeThisQueryParser.java

```diff
@@ -116,37 +108,29 @@ public Query parse(QueryShardContext context) throws IOException, QueryParsingEx
                 } else if (parseContext.parseFieldMatcher().match(currentFieldName, Field.LIKE_TEXT)) {
                     likeTexts.add(parser.text());
                 } else if (parseContext.parseFieldMatcher().match(currentFieldName, Field.MAX_QUERY_TERMS)) {
-                    mltQuery.setMaxQueryTerms(parser.intValue());
+                    maxQueryTerms = parser.intValue();
                 } else if (parseContext.parseFieldMatcher().match(currentFieldName, Field.MIN_TERM_FREQ)) {
-                    mltQuery.setMinTermFrequency(parser.intValue());
+                    minTermFreq =parser.intValue();
                 } else if (parseContext.parseFieldMatcher().match(currentFieldName, Field.MIN_DOC_FREQ)) {
-                    mltQuery.setMinDocFreq(parser.intValue());
+                    minDocFreq = parser.intValue();
                 } else if (parseContext.parseFieldMatcher().match(currentFieldName, Field.MAX_DOC_FREQ)) {
-                    mltQuery.setMaxDocFreq(parser.intValue());
+                    maxDocFreq = parser.intValue();
                 } else if (parseContext.parseFieldMatcher().match(currentFieldName, Field.MIN_WORD_LENGTH)) {
-                    mltQuery.setMinWordLen(parser.intValue());
+                    minWordLength = parser.intValue();
                 } else if (parseContext.parseFieldMatcher().match(currentFieldName, Field.MAX_WORD_LENGTH)) {
-                    mltQuery.setMaxWordLen(parser.intValue());
+                    maxWordLength = parser.intValue();
                 } else if (parseContext.parseFieldMatcher().match(currentFieldName, Field.ANALYZER)) {
-                    analyzer = context.analysisService().analyzer(parser.text());
+                    analyzer = parser.text();
                 } else if (parseContext.parseFieldMatcher().match(currentFieldName, Field.MINIMUM_SHOULD_MATCH)) {
-                    mltQuery.setMinimumShouldMatch(parser.text());
+                    minimumShouldMatch = parser.text();
                 } else if (parseContext.parseFieldMatcher().match(currentFieldName, Field.BOOST_TERMS)) {
-                    float boostFactor = parser.floatValue();
-                    if (boostFactor != 0) {
-                        mltQuery.setBoostTerms(true);
-                        mltQuery.setBoostTermsFactor(boostFactor);
-                    }
-                } else if (parseContext.parseFieldMatcher().match(currentFieldName, Field.MINIMUM_SHOULD_MATCH)) {
-                    mltQuery.setMinimumShouldMatch(parser.text());
-                } else if ("analyzer".equals(currentFieldName)) {
```

Comment:
I noticed that too. There are not duplicates in master but they are in the query refactoring branch. Probably something that happened during merge.